standard/standard_7.php

```diff
@@ -47,20 +47,34 @@
  * fwrite, fclose, and
  * feof). If the call fails, it will return false
  */
-function fsockopen(string $hostname, int $port = -1, &$error_code, &$error_message, ?float $timeout) {}
+function fsockopen(
+    string $hostname,
+    #[PhpStormStubsElementAvailable(from: '5.3', to: '7.0')] int $port,
+    #[PhpStormStubsElementAvailable(from: '7.1')] int $port = -1,
+    &$error_code,
+    &$error_message,
+    ?float $timeout
```

Comment:
`$error_code`, `$error_message`, `$timeout` have default value `null` since 8.0.

Reply:
The answer is the same as above. let's leave it as is as they are already marked as optional